src/usage_metrics/core/eel_hole.py

```diff
@@ -0,0 +1,103 @@
+"""Transform data from viewer.catalyst.coop logs."""
+
+import json
+import os
+from urllib.parse import urlsplit
+
+import pandas as pd
+from dagster import (
+    AssetExecutionContext,
+    WeeklyPartitionsDefinition,
+    asset,
+)
+
+from usage_metrics.helpers import convert_camel_case_columns_to_snake_case
+
+
+@asset(
+    partitions_def=WeeklyPartitionsDefinition(start_date="2023-08-16"),
+    # io_manager_key="database_manager",
+    tags={"source": "eel-hole"},
+)
+def core_eel_hole_logs(
+    context: AssetExecutionContext,
+    raw_eel_hole_logs: pd.DataFrame,
+) -> pd.DataFrame:
+    """Transform viewer.catalyst.coop logs."""
+    context.log.info(f"Processing data for the week of {context.partition_key}")
+
+    if raw_eel_hole_logs.empty:
+        context.log.warn(f"No data found for the week of {context.partition_key}")
+        return raw_eel_hole_logs
+
+    # Flatten the many nested columns
+    json_df = pd.json_normalize(raw_eel_hole_logs.jsonPayload).rename(
+        columns={"timestamp": "json_timestamp"}
+    )  # Rename to avoid duplicate column names
+    labels_df = pd.json_normalize(raw_eel_hole_logs.labels)
+    df = raw_eel_hole_logs.drop(
+        columns=["jsonPayload", "labels", "resource"]
+    )  # We just don't care about 'resource', which tracks some cloud logging stuff we'd prefer to monitor in GCS directly
+
+    df = pd.concat([df, json_df, labels_df], axis=1)
```

Comment:
this naked (aka non-restircted) concat here gives me slight heebie jeebies.  there are instances of many `json_df` records per each `df` record... which would be fine i think. but are there sometimes any instances of `labels_df`?  which would mean duplicating in a many to many way when there are 1:m df:json and 1:m df:labels.